packages/appstore/src/components/modals/real-wallets-upgrade/components/wallet-linking-step/wallet-linking-step.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { Text, ThemedScrollbars } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { observer, useStore } from '@deriv/stores';
+import WalletLinkWrapper, { TWalletLinkWrapper } from '../wallet-link/wallet-link-wrapper';
+import './wallet-linking-step.scss';
+
+type TWalletLinkingStep = {
+    data: { title: string; wallets: TWalletLinkWrapper[] };
+};
+
+const WalletLinkingStep = observer(({ data }: TWalletLinkingStep) => {
+    const { ui } = useStore();
+    const { is_mobile } = ui;
+    return (
+        <div className='wallet-linking-step'>
+            <Text as='div' color='prominent' size={is_mobile ? 's' : 'm'} weight='bold'>
+                {data.title}
+            </Text>
+            <Text as='div' className='wallet-linking-step__description' color='prominent' size={is_mobile ? 'xs' : 's'}>
+                {localize('This is how we link your accounts with your new Wallet.')}
+            </Text>
+            <Text
+                as='span'
+                className='wallet-linking-step__note wallet-linking-step__title-text'
+                color='prominent'
+                size='xxs'
+            >
+                {localize(
+                    'Your existing funds will remain in your trading account(s) and can be transferred to your Wallet after the upgrade.'
+                )}
+            </Text>
+            {!is_mobile && (
```

Comment:
shouldn't we use isDesktop here?

Reply:
Since we won't be using utility functions from shared component and we only have is_mobile flag in the store.